src/services/user.service.ts

```diff
@@ -0,0 +1,24 @@
+import { UserRepository } from '../repository/user.repository';
+import { User } from '../entity/user.entity';
+import { getCustomRepository } from 'typeorm';
+/**
+ * Cette classe est un service
+ * C'est ici que l'ensemble de la logique consernant les psort doit apparaitre.
+ * Attention ! Mettez le moins possible d'element dans le controlleur
+ */
+export class UserService {
+
+
+    private repository = getCustomRepository(UserRepository);
+
+    // Business logic
+    async getAll() {
+        return await this.repository.find();
+    }
+
+    async insert(user: any) {
+        await this.repository.create(user);
```

Comment:
Le create dans se contexte ne sert a rien. Pour que celui-ci ait un impacte, il faut que tu écrives: 
`user = await this.repository.create(user);`
Mais plus important, le create n'est pas utile dans le cas d'utilisation actuelle car tu récupère le customRepository UserRepository. C'est donc un repo déjà config pour le user. Pas besoin de faire un create ou un new User comme il est déjà config pour recevoir un user